### PR TITLE
Aggregated NCDA - entry point + result table

### PR DESCRIPTION
**Pull request by anvmn**
#705

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2023-05-04 16-24

**Comment by anvmn** [comment]
Peek 2023-05-05 00-16

**Comment by anvmn** [comment]
Selection_1827

**Comment by anvmn** [comment]
Selection_1829

File: server/elm/src/Pages/Scoreboard/View.elm [hunk 1]
```diff
@@ -1,23 +1,536 @@
...
+            emptyNode
+
+
+viewScoreboardData : Language -> NominalDate -> ScoreboardData -> Model -> Html Msg
+viewScoreboardData language currentDate data model =
+    let
+        topBar =
+            div [ class "top-bar" ]
+                [ div [ class "new-selection" ]
+                    [ a [ href "/admin/reports/aggregated-ncda" ]
```
Comment (AronNovak) [inline server/elm/src/Pages/Scoreboard/View.elm:42]: Where do we check if this given user has access to that URL? Or is it accessible for all users?

Comment (anvmn) [reply server/elm/src/Pages/Scoreboard/View.elm:42]: This is Drupal page, so access check is on Drupal side.

File: server/elm/src/Pages/Scoreboard/View.elm [hunk 2]
```diff
@@ -1,23 +1,536 @@
...
+                (\item itemValues ->
+                    viewTableRow language currentDate yearSelectorGap (Translate.NCDADemographicsItemLabel item) itemValues
+                )
+                [ ChildrenUnder2, NewbornsThisMonth, LowBirthWeigh ]
+                values
+
+        values =
+            case entityType of
+                EntityVillage ->
+                    [ [ 12, 12, 14, 13, 15, 15, 15, 12, 13, 13, 14, 14 ]
```
Comment (AronNovak) [inline server/elm/src/Pages/Scoreboard/View.elm:116]: at first sight, it"s not obvious that what are these

Comment (anvmn) [reply server/elm/src/Pages/Scoreboard/View.elm:116]: These are demo values - TIP team needed to present this feature, so they asked to have it.
Actual values are generated at follow up PRs.

File: server/elm/src/Pages/Scoreboard/View.elm (hunk 1 above)
Comment (AronNovak) [reply server/elm/src/Pages/Scoreboard/View.elm:42]: Yep, but can it happen that we display this link, but they get a 403?

Comment (anvmn) [reply server/elm/src/Pages/Scoreboard/View.elm:42]: But it's good that you mentioned it, as I now see that a bug I fixed at https://github.com/TIP-Global-Health/eheza-app/pull/706/commits/c0b1e4280116555448a71973577937eec7cb3d36 reapeared again.

Fixed it at [06003d1](https://github.com/TIP-Global-Health/eheza-app/pull/706/commits/06003d17dc6cc8dc06f551864fbd9cbc5662cf37).

Comment (anvmn) [reply server/elm/src/Pages/Scoreboard/View.elm:42]: Well, all pages got same access check. So, if you're on page that displays the link, you can access the link.

**Comment by anvmn** [comment]
@AronNovak ?

**Review by AronNovak — APPROVED** [review]

**Pull request merged**